qa/README.md

```diff
@@ -13,16 +13,17 @@ Unix
 ----
 `python3-zmq` and `ltc_scrypt` are required. On Ubuntu or Debian they can be installed via:
 ```
-sudo apt-get update && apt-get install -y curl python3-zmq python3-dev gcc
-cd qa/pull-tester/ && ./install-deps.sh
+sudo apt-get update
+sudo apt-get install -y curl python3-zmq python3-dev gcc
```

Comment:
does this need `python3-setuptools` too?

Reply:
i've updated the pertinent sections in ci that run qa to keep `python3-zmq` and install `python3-pip` as the recommended packages installed on `bionic` are:
```
root@2ad4e01d2056:~/dogecoin# apt-get install python3-pip
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  dh-python python3-asn1crypto python3-cffi-backend python3-crypto python3-cryptography python3-dbus
  python3-dev python3-distutils python3-gi python3-idna python3-keyring python3-keyrings.alt python3-lib2to3
  python3-pkg-resources python3-secretstorage python3-setuptools python3-six python3-wheel python3-xdg
Suggested packages:
  python-crypto-doc python-cryptography-doc python3-cryptography-vectors python-dbus-doc python3-dbus-dbg
  gnome-keyring libkf5wallet-bin gir1.2-gnomekeyring-1.0 python-secretstorage-doc python-setuptools-doc
The following NEW packages will be installed:
  dh-python python3-asn1crypto python3-cffi-backend python3-crypto python3-cryptography python3-dbus
  python3-dev python3-distutils python3-gi python3-idna python3-keyring python3-keyrings.alt python3-lib2to3
  python3-pip python3-pkg-resources python3-secretstorage python3-setuptools python3-six python3-wheel
  python3-xdg
```
will have to update this at `bionic` eol anyway but works locally for time being. waiting for ci to confirm, otherwise will add required packages explicitly.

Reply:
hmm seems recommended packages were not installed so i'll have to add explicitly and re-run. feel free to cancel the ci run if you want as it's failed.